components/Mappings/Mappings.js

```diff
@@ -733,26 +739,19 @@ class Mappings extends Component {
 					errorMessage: 'Unable to update Synonyms',
```

Comment:
Ideally, we should clear error related state from the context if we are handling error modals directly from the response handler.